auth-web/src/store/modules/org.ts

```diff
@@ -420,6 +428,42 @@ export default class OrgModule extends VuexModule {
     }
   }
 
+  @Action({ commit: 'setCurrentOrgTransactionList', rawError: true })
+  public async getTransactionList (dateFilter: TransactionDateFilter) {
+    const response = await PaymentService.getTransactions(this.context.state['currentOrganization'].id, dateFilter)
+    if (response?.data) {
+      const formattedList = await this.formatTransactionTableData(response.data.items || [])
+      return {
+        limit: response.data.limit,
+        page: response.data.page,
+        total: response.data.total,
+        transactionsList: formattedList
+      }
+    }
+    return {}
+  }
+
+  @Action({ rawError: true })
+  public formatTransactionTableData (transactionList) {
```

Comment:
do you mind writing some comments on this ? the code is great.. if there are some comments , will be easy to understand the code easily..